server/routerlicious/packages/lambdas/src/scribe/utils.ts

```diff
@@ -51,3 +53,43 @@ export const sendToDeli = (
 export const getClientIds = (protocolState: IProtocolState, clientCount: number) => {
 	return protocolState.members.slice(0, clientCount).map((member) => member[0]);
 };
+
+/**
+ * Whether to write checkpoint to local db.
+ * @param noActiveClients - whether there are any active clients
+ * @param globalCheckpointOnly - whether to always write checkpoints to global db
+ * @returns whether to write checkpoint to local db
+ */
+export const isLocalCheckpoint = (noActiveClients: boolean, globalCheckpointOnly: boolean) => {
+	return !isGlobalCheckpoint(noActiveClients, globalCheckpointOnly);
+};
+/**
+ * Whether to write checkpoint to global db.
+ * @param noActiveClients - whether there are any active clients
+ * @param globalCheckpointOnly - whether to always write checkpoints to global db
+ * @returns whether to write checkpoint to global db
+ */
+export const isGlobalCheckpoint = (noActiveClients: boolean, globalCheckpointOnly: boolean) => {
+	return noActiveClients || globalCheckpointOnly;
+};
+
+/**
+ * Whether the quorum members represented in the checkpoint's protocol state are valid.
```

Comment:
Nit: I wonder if we should should call this isCheckpointQuorumScrubbed since we are not checking for validity of the quorum beyond whether the data has been scrubbed or not